packages/images-plugin/lib/finder.js

```diff
@@ -224,8 +220,10 @@ export default class ImageElementFinder {
     }
 
     if (_.isEmpty(results)) {
-      if (multiple) {
-        return [];
+      if (multiple === true) {
```

Comment:
why `if (multiple)` would not work?

Reply:
it might; I think I changed this trying to fix something else.